scripts/verify-agent-memory.sh

```
@@ -73,6 +78,81 @@ for required_contract in "${required_agent_contracts[@]}"; do
+    '        --quality low \' \
+    '        --size 1024x1024 \' \
+    '        --no-augment \' \
+    '        --out "$output_file"' \
+    ')')
+
+if [[ "$openrouter_reference" != *"$required_openrouter_runtime_block"* ]]; then
+    fail "OpenRouter image-generation reference is missing the exact isolated runtime block"
+fi
+
+forbidden_openrouter_contracts=(
+    'OPENAI_API_KEY="$OPENROUTER_API_KEY"'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)